src/simulator_core/infrastructure/app.py

```diff
@@ -33,8 +34,10 @@ def run(file_path: str | None = None) -> pd.DataFrame:
     config = SimulationConfiguration(simulation_id=uuid.uuid1(),
                                      name="test run",
                                      timestep=3600,
-                                     start=0,
-                                     stop=3600 * 1)
+                                     start=datetime.strptime("2019-01-01T00:00:00",
```

Comment:
Dit is een voorlopige definitie toch?

Reply:
Ja, zodat het nu werkt. Ik weet nog niet wat we precies van bovenaf gaan doorkrijgen